cli/popper/cli.py

```diff
@@ -96,4 +97,8 @@ def signal_handler(sig, frame):
         pu.info("Stopping container '{}'\n".format(container.name))
         container.remove(force=True)
 
+    if len(sclient.instances()) > 0:
+        pu.info("Stopping all instances\n")
+        sclient.instance_stopall()
```

Comment:
@ivotron Yes, a container runs as an instance and gets removed on its own after `run` or `exec` completes execution automatically. Thus a short lived instance is created. So what shall we do on `SIGINT` since its already gets handled by singularity and it cleans up running container on its own ?

Reply:
If `singularity-cli` is already handling `SIGINT` accordingly, then there's nothing we need to do here. Do you know if this is the case for `docker-py` as well? If that's the case, then we could also remove the code that stops docker containers, and only deal with processes started by `utils.exec_cmd`